packages/sdk/src/handlers/tfa.ts

```diff
@@ -0,0 +1,17 @@
+import { ITransport } from '../transport';
+
+export class TFAHandler {
+	private transport: ITransport;
+
+	constructor(transport: ITransport) {
+		this.transport = transport;
+	}
+
+	async enable(password: string): Promise<void> {
+		await this.transport.post('/users/tfa/enable', { password });
+	}
+
+	async disable(otp: string): Promise<void> {
+		await this.transport.post('/users/tfa/disable', { otp });
+	}
```

Comment:
@WoLfulus Can we update this to match https://github.com/directus/directus/pull/4678 ?